core/templates/dev/head/services/contextual/logger.service.spec.ts

```diff
@@ -0,0 +1,62 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for LoggerService.
+ */
+import { TestBed } from '@angular/core/testing';
+import { LoggerService } from 'services/contextual/logger.service';
+
+describe('Logger Service', () => {
+  let ls;
+
+  beforeEach(() => {
+    ls = TestBed.get(LoggerService);
+  });
+
+  it('should console a debug message', () => {
```

Comment:
should display debug message on the console

Here and below, please use this kind of statement.

Reply:
Done.